.github/workflows/sandbox-build-deploy.yml

```diff
@@ -1,14 +1,54 @@
-name: CI
-on: [push]
+name: CICD
+on: 
+  push: 
+    branches:
+      - qa
 jobs:
   build:
-    name: Test
+    name: build-deploy-to-sandbox
+    environment: qa
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: borales/actions-yarn@v2.3.0
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install Dependency
+        uses: borales/actions-yarn@v2.3.0
         with:
           cmd: install # will run `yarn install` command
-      - uses: borales/actions-yarn@v2.3.0
+      - name:  Build
+        uses: borales/actions-yarn@v2.3.0
         with:
           cmd: build # will run `yarn build` command
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SSH_PRIVATE_KEY }}
+          known_hosts: ${{ secrets.SSH_HOST }}
+      - name: Adding known hosts
+        run: ssh-keyscan -H ${{ secrets.SSH_HOST }} >> ~/.ssh/known_hosts          
+      - name: Copy Package
+        run: |
+          rm -rf prod-build
+          mkdir prod-build
+          mkdir prod-build/modules
+          cp -r  packages/core/build prod-build/modules/core
+          cp -r  packages/classes/build prod-build/modules/classes
+          cp -r  packages/attendance/build prod-build/modules/attendance
+          cp -r  packages/timetable/build prod-build/modules/timetable
+          cp -r  packages/worksheet/build prod-build/modules/worksheet
+          cp -r  packages/student/build prod-build/modules/student
```

Comment:
hey @arajput can't we automate this, each time with module we have to add it here?

Reply:
yes that is pending item.  i  will create another ticket to fix this. it will  require some time

Reply:
@Shruti3004   - this is fixed now.  added a  bash script to iterate over packages and copy  into prod-build except common-lib and teacher-app. teacher-app  is copied to root directory explicitly